lib/src/widgets/reorderable_sliver.dart

```diff
@@ -988,8 +994,10 @@ class _ReorderableSliverListState extends State<ReorderableSliverList>
       transform: Matrix4.rotationZ(0),
       alignment: FractionalOffset.topLeft,
       child: Material(
-        child:
-            Card(child: ConstrainedBox(constraints: constraints, child: child)),
+        child: Card(
+          child: ConstrainedBox(constraints: constraints, child: child),
+          color: Colors.transparent,
```

Comment:
Instead of hard-coding a specific color, you should inject it as a property of `ReorderableSliverList`. Something like `feedbackBackgroundColor`, which by default can be `Colors.white` in order to not break things, but can be overridden very easily with a custom value.